Quickstart.md

```diff
@@ -0,0 +1,197 @@
+### Setup the appcli enironment
+
+#### Create the required directory structure
+
+Run the following commands to create the required files and folders
+
+```
+mkdir -p src/resources/templates/{baseline,configurable}
+touch src/resources/myapp.py
+touch src/resources/settings.py
+touch src/resources/stack-settings.py
+```
+
+
+Navigate to the newly created myapp.py file. Paste in the folllowing code
+
+```python
+#!/usr/bin/env python3
+# # -*- coding: utf-8 -*-
+
+# filename: myapp.py
+
+# standard libraries
+import os
+import sys
+from pathlib import Path
+
+# vendor libraries
+from appcli.cli_builder import create_cli
+from appcli.models.configuration import Configuration
+from appcli.orchestrators import DockerComposeOrchestrator
+
+# ------------------------------------------------------------------------------
+# CONSTANTS
+# ------------------------------------------------------------------------------
+
+# directory containing this script
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+
+# ------------------------------------------------------------------------------
+# PRIVATE METHODS
+# ------------------------------------------------------------------------------
+
+def main():
+    configuration = Configuration(
+        app_name='myapp',
+        docker_image='brightsparklabs/myapp',
+        seed_app_configuration_file=Path(BASE_DIR, 'resources/settings.yml'),
+        stack_configuration_file=Path(BASE_DIR, 'resources/stack-settings.yml'),
+        baseline_templates_dir=Path(BASE_DIR, 'resources/templates/baseline'),
+        configurable_templates_dir=Path(BASE_DIR, 'resources/templates/configurable'),
+        orchestrator=DockerComposeOrchestrator(
+            docker_compose_file = Path('docker-compose.yml')
+        ),
+        
+    )
+    cli = create_cli(configuration)
+    cli()
+
+# ------------------------------------------------------------------------------
+# ENTRYPOINT
+# ------------------------------------------------------------------------------
+
+if __name__ == '__main__':
+    main()
+```
+
+Make sure the myapp.py has executable permission. In order to do this, you need to cd into the directory of myapp.py in your terminal and execute the following command
+
+```
+chmod +x myapp.py
+```
+
+### Dockerfile Setup
+
+Outside of the src folder, create a docker file and paste in the following settings so we can assemble our image
+
+```Dockerfile
+# filename: Dockerfile
+
+FROM brightsparklabs/appcli
+
+ENTRYPOINT ["./myapp.py"]
+WORKDIR /app
+
+# install compose if using it as the orchestrator
+RUN pip install docker-compose
+
+COPY requirements.txt .
+RUN pip install --requirement requirements.txt
+COPY src .
+
+ARG APP_VERSION=latest
+ENV APP_VERSION=${APP_VERSION}
+```
+
+We can see in the dockerfile, it is referring to a requirements.txt file which we will need to create. Go ahead and create a requirements.txt file in the same location as your Dockerfile and enter in the following `bsl-appcli==1.3.4` 
```

Comment:
dockerfile to `Dockerfile` (also wrap in backticks)
`requirements.txt` in backticks